lib/managers/webhooks.js

```diff
@@ -194,6 +279,30 @@ Webhooks.prototype.delete = function(webhookID, callback) {
 };
 
 /**
+ * Validate a webhook message by verifying the signature and the delivery timestamp
+ *
+ * @param {string} body - The request body of the webhook message
+ * @param {Object} headers - The request headers of the webhook message
+ * @param {string} primarySignatureKey - The primary signature to verify the message with
+ * @param {string} [secondarySignatureKey] - The secondary signature to verify the message with
+ * @param {int} [maxMessageAge] - The maximum message age (in seconds).  Defaults to 10 minutes
+ * @returns {boolean} - True or false
+ */
+Webhooks.validateMessage = function(body, headers, primarySignatureKey, secondarySignatureKey, maxMessageAge) {
+	if (!validateSignature(body, headers, primarySignatureKey, secondarySignatureKey)) {
+		return false;
+	}
+
+	if (!validateDeliveryTimestamp(headers, maxMessageAge | MAX_MESSAGE_AGE)) {
```

Comment:
I used `(maxMessageAge === null) ? MAX_MESSAGE_AGE : maxMessageAge` since I didn't want a non-numeric value to get silently ignored (should result in an error instead).